v2/tele/tele.go

```diff
@@ -129,6 +130,6 @@ func FromContext(ctx context.Context, attrs ...attribute.KeyValue) attribute.Set
 }
 
 // StartSpan creates a span and a [context.Context] containing the newly-created span.
-func StartSpan(ctx context.Context, name string) (context.Context, trace.Span) {
-	return otel.Tracer(TracerName).Start(ctx, name)
+func StartSpan(ctx context.Context, name string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
```

Comment:
Actually I think we should remove this function. It seemed like a good idea but it's better to use the embedded tracers rather than the global one

Reply:
Would be good to remove as part of this cleanup

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/issues/new